onnxruntime/contrib_ops/webgpu/bert/flash_attention.wgsl.template

```diff
@@ -323,6 +345,27 @@ $MAIN {
   if (valid_q) {
     writeo(q_idx_global, head_idx, local_idx);
   }
+#else
+#if is_nvidia
+    for (var i : u32 = 0; i < head_size_vec; i++) {
+      var sum = v_tile[0][i] * qk_1[0];
```

Comment:
add a wrapper function fetchVTile(i) that either uses subgroupshuffle or broadcast access.
This way lines 350-368 can be the same as lines 372 to 388.